src/main/java/com/mercadolivre/bootcamp/projeto_integrador/exception/SectionNotFound.java

```diff
@@ -0,0 +1,12 @@
+package com.mercadolivre.bootcamp.projeto_integrador.exception;
+
+import org.springframework.http.HttpStatus;
+
+import java.time.ZonedDateTime;
+
+public class SectionNotFound extends BaseException{
+
+    public SectionNotFound(String message, HttpStatus httpStatus, ZonedDateTime timestamp) {
+        super(message, httpStatus, timestamp);
```

Comment:
Exception incompleta. Que tal definir a mensagem padrão da exception aqui assim como feito em SampleException?

```suggestion
        super("A section com ID: " + message + " não foi encontada", HttpStatus.STATUS_PADRÃO_QUE_VOCE_QUER_ENVIAR, ZonedDateTime.now());
```

Dessa forma toda SectionNotFoundException lançada terá o mesmo status code, formato e regra de negócio para horário.